.github/workflows/ci.yaml

```diff
@@ -13,30 +13,24 @@ env:
 
 jobs:
   rustfmt:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@nightly-2022-07-22
```

Comment:
Suggest this be `@master` with the rust toolchains listed in a strategy matrix.

Reply:
used master with matrix strategy